### ISSUE TITLE
Is there any way to support human in the loop

### ISSUE THREAD

**Issue by SaihholdChiu** [openvinotoolkit/cvat#3329]
I’m using CVAT to do an object detection labeling, my method is to label 1000 images and download to train my model , then upload predictions to CVAT and fix, do it again and again , is there any way to do all these steps in one platform?

**Comment by nmanovic** [comment]
@SaihholdChiu , it is something which we want to implement in the nearest future.

**Comment by doantientai** [comment]
Hi @nmanovic, do we have any news on this? Thank you!

**Comment by doantientai** [comment]
By the way, Label Studio has a workaround for this, which is integrating a model to the system so that we can train/inference it during the labeling process. Thanks to the prediction score, we sort the samples to annotate the ones with low confidence first.

Can we do something similar?

**Comment by nikste** [comment]
any update on this ? 
I am also highly interested. As far as I'm aware SAM (and SAM2) were trained with a similar way.
There are some tutorials about finetuning SAM2 on custom data, that could be used for this.
e.g.: https://towardsdatascience.com/train-fine-tune-segment-anything-2-sam-2-in-60-lines-of-code-928dd29a63b3

**Comment by nmanovic** [comment]
@nikste , @doantientai , we are working on a general approach for that. I hope to release something this year.

**Comment by sarasilva-digitalsense** [comment]
Are there any updates on this?
